### ISSUE TITLE
Constant "Browser failed to start in time" issue

### ISSUE THREAD

**Issue by Alex-Sh** [sitespeedio/sitespeed.io#1510]
Hi

We are getting "Browser failed to start in time" issue with both browsers.
Issue appears quite constant in a random time and run. If it will appear on first run, there will be no other runs like in example below.

_Note, we are trying to execute 3 "docker run" in parallel to speed up check process. But issue is appearing even with single execution_

Command: 
`docker run --shm-size=512m --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io:4.5.1 -b firefox -n 3 --browsertime.delay=1000 --graphite.namespace=sitespeed_io.monitoring --graphite.host=xxx`

Output
```
Google Chrome 57.0.2987.74 beta
Mozilla Firefox 51.0.1
[2017-03-12 13:06:49] Versions OS: linux 3.10.0-514.6.1.el7.x86_64 nodejs: v6.9.1 sitespeed.io: 4.5.1 browsertime: 1.0.0-beta.28 coach: 0.31.0
[2017-03-12 13:06:57] Starting firefox for analysing http://xxx 3 time(s)
[2017-03-12 13:06:59] Testing url http://xxx run 1
[2017-03-12 13:07:59] Browser failed to start in time, trying one more time.
[2017-03-12 13:09:00] Render HTML for 1 page(s) 
[2017-03-12 13:09:04] HTML stored in /sitespeed.io/sitespeed-result/xxx/2017-03-12-13-06-49
[2017-03-12 13:09:04] Finished analysing http://xxx
```
`Linux version 3.10.0-514.6.1.el7.x86_64 (builder@kbuilder.dev.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-11) (GCC) ) #1 SMP Wed Jan 18 13:06:36 UTC 2017, MemTotal:  12277908 kB `

Thanks

**Comment by soulgalore** [comment]
Hi @Alex-Sh 
generally I use to run one browser at the time to make sure nothing else can influence the metrics. 

How does the CPU/memory looks like when you run multiple instances?  "Browser failed to start in time" often is the problem when running on a to slow instance (for example it can happen when we run in Travis). I think I never seen it on Firefox before. 

This looks like a bug on our side, you should get another log saying it fails the second time, so it's more clear that it didn't work.
<pre>
[2017-03-12 13:06:59] Testing url http://xxx run 1
[2017-03-12 13:07:59] Browser failed to start in time, trying one more time.
[2017-03-12 13:09:00] Render HTML for 1 page(s) 
</pre>

Best
Peter

**Comment by soulgalore** [comment]
Hey @Alex-Sh was you able to make it work for you?

**Comment by Alex-Sh** [comment]
Hi, no, will see how it goes on production server.
But I'm not sure it's about server performance. 3 parallel runs it's not so huge.

**Comment by soulgalore** [comment]
ok, I would say it depends on the server and the page you are testing :)

**Comment by soulgalore** [comment]
@Alex-Sh do you have any updates?

**Comment by beenanner** [comment]
Closing for now, but feel free to open a new issue if this is still occurring with latest version of sitespeed.io. We would probably need server specs, example URLs, how often they are tested, etc for steps to reproduce and help debug.
